barretenberg/cpp/src/barretenberg/execution_trace/execution_trace.cpp

```diff
@@ -153,4 +153,4 @@ template class ExecutionTrace_<MegaFlavor>;
 template class ExecutionTrace_<plonk::flavor::Standard>;
 template class ExecutionTrace_<plonk::flavor::Ultra>;
 
-} // namespace bb
\ No newline at end of file
+} // namespace bb
```

Comment:
<!--__GRAPHITE_HTML_TAG_START__--><i class='graphite__hidden'>[Re: line +157]<br/><br/></i>

<!--__GRAPHITE_HTML_TAG_END__-->nit: change not needed<!--__GRAPHITE_HTML_TAG_START__--><span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/AztecProtocol/aztec-packages/8535?utm_source=unchanged-line-comment">Graphite</a>.</span>

<!--__GRAPHITE_HTML_TAG_END__-->